packages/stream-model/src/model.ts

```diff
@@ -155,12 +156,14 @@ export class Model extends Stream {
 
   private _isReadOnly = false
 
-  get content(): ModelDefinition {
+  override get content(): ModelDefinition {
     return super.content
   }
 
   get metadata(): ModelMetadata {
-    return { controller: this.state$.value.metadata.controllers[0], model: Model.MODEL }
+    const controller = this.state$.value.metadata.controllers[0]
+    if (!controller) throw new Error(`Controller must be present`)
```

Comment:
same comment as from the other PR on model-instance-document, this should always be set